src/config.inc

```diff
@@ -5,7 +5,7 @@ BUILD_ENV = AUTO
 ifeq ($(BUILD_ENV),MSVC)
   #CXXFLAGS += /Wall /WX
 else
-  CXXFLAGS += -Wall -pedantic -Werror -Wno-deprecated-declarations -Wswitch-enum
+  CXXFLAGS += -Wall -pedantic -Wno-deprecated-declarations -Wswitch-enum
```

Comment:
I take it that this is because a new compiler version warns about things the older ones did not? Can we use `-Wno-error=warning` to only disable these new warnings being treated as errors? Also should be in a separate commit.

Reply:
This is an accident actually, I need this change only locally because I’m using some newer compiler versions than what we’re running on CI.